### PR TITLE
KOGITO-1868 - Add Zulip notification to daily builds

### PR DESCRIPTION
**Pull request by cristianonicolai**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by radtriste** [comment]
This will create a new topic each time ?

**Comment by cristianonicolai** [comment]
topic per subject as is now: 
`Kogito daily snapshot deploy #${BRANCH_NAME}: ${currentBuild.currentResult}`

**Comment by cristianonicolai** [comment]
@radtriste 
Reduced to:

- `Kogito daily snapshot deploy #${BRANCH_NAME}`
- `Kogito daily Quarkus master`

File: Jenkinsfile.quarkus [hunk 1]
```diff
@@ -60,23 +60,17 @@ pipeline {
...
-        failure{
-            emailext body: 'kogito-quarkus #${BUILD_NUMBER} was: ' + "${currentBuild.currentResult}" +  '\\n' +
-                    'Please look here: ${BUILD_URL} \\n' +
-                    ' \\n' +
-                    '${BUILD_LOG, maxLines=750}', subject: 'kogito quarkus #${BUILD_NUMBER}: ' + "${currentBuild.currentResult}", to: 'cnicolai@redhat.com, mbiarnes@redhat.com'
+        failure {
+            emailext body: 'Kogito daily Quarkus master #${BUILD_NUMBER} was: ' + "${currentBuild.currentResult}" +  '\n' +
+                     'Please look here: ${BUILD_URL} ',
+                     subject: 'Kogito daily Quarkus master',
+                     to: 'kogito-ci.f581f6cfb146dbcae96fa8839c768066.prefer-text@streams.zulipchat.com, mbiarnes@redhat.com'
```
Comment (ricardozanini) [inline Jenkinsfile.quarkus:67]: This one @cristianonicolai :heart:

**Pull request merged**